get_status.py

```
@@ -0,0 +1 @@
+import sys, json; data=json.load(sys.stdin); [print(f"{s[\"context\"]}: {s[\"state\"]} - {s[\"target_url\"]}") for s in data.get("statuses", []) if s["state"] != "success"]
```

Ruff (0.15.21)

[error] Multiple statements on one line (semicolon)

(E702)